content/post/writing-a-book-remotely.md

```diff
@@ -0,0 +1,576 @@
+
+While we had no strict roadmap in mind, we all agreed to keep this book up-to-date to avoid a stale book.
+
+Whenever there's a new update for our eBook we also send out a small release information email to our newsletter to give a short overview of the changes:
+
+![Stratospheric Post Release Mailing List Update](/img/stratospheric-ebook/stratospheric-post-release-mailing-list-update.png#center "Stratospheric Post Release Mailing List Update")
+
+On the monetary side of things: We don't get rich from selling this book. It's some additional pocket money for us in the range of $150-300 for every author each month.
+
+To increase the visibility of our eBook we listed it on Amazon Kindle. While Amazon's royalty rules are not that optimal compared to Leanpub, we primarily list on Amazon to get in front of more developers. Amazon is one of the biggest search interfaces after Google and YouTube.
+
+We thought about a printed eBook version but for now postponed this move as we think that our audience is anyway more interested to read technical books on their devices. However, (at least I) want to have a physical copy of this book in my bookshelf somewhere in the future.
+
+Based on the feedback we got so far from our readers, we also decided to build an online course on top of the eBook to provide some more in-depth coverage of various areas. Given my [existing experience with creating and launching technical online courses](https://progmot.com/post/creating-and-selling-my-first-online-course/), this will be a great opportunity to apply my lessons learned from my first courses.
+
+Getting to the end of the book, some publishers started to reach out to us. They wanted to publish Stratospheric from their publishing house and bring it to various marketplaces. We . The additional outreach may be great but the royalties are diminishing compared to Leanpub, especially if we have to split them into three equal parts. To stay independent we chose against a publisher and published the eBook ourselves on Amazon to have at least the big search traffic from this major ecommerce platform. We know that Leanpub is a niche site and not all developers are available of this great source of technical books. However, given our existing audience from our blog, newsletter, Twitter, etc. we make our potential readers aware via those channels.
+
+At the time of writing this article, we have 800 readers, which is a great success 🥳
+
+## Recommendations, Tips & Tricks, and Lessons Learned 🎩
+
+What follows is an unordered list of hints and tips about this journey. If you plan to start a similar project, make sure to learn from our _mistakes_ and recommendations.
+
+### Write Together, Not Alone
+
+Writing a book together has a lot of benefits. It motivates you to keep going when you would otherwise quite when working on your own.
```

Comment:
```suggestion
Writing a book together has a lot of benefits. It motivates you to keep going when you would otherwise have quite when working on your own.
```